doc/modules/ROOT/pages/index.adoc

```diff
@@ -1,46 +1,154 @@
+pod/hazelcast-1   1/1     Running   0          2m11s
+pod/hazelcast-2   1/1     Running   0          2m1s
+
+NAME                 TYPE           CLUSTER-IP       EXTERNAL-IP     PORT(S)          AGE
+service/hazelcast    LoadBalancer   10.171.245.205   35.240.109.13   5701:31092/TCP   2m22s
+service/kubernetes   ClusterIP      10.171.240.1     <none>          443/TCP          38m
+
+NAME                         READY   AGE
+statefulset.apps/hazelcast   3/3     2m22s
+----
+
+The external IP of the Hazelcast cluster is: *35.240.109.13*.
+
+== 2. Create Publisher Cluster
+
+Again, we need to grant Kubernetes API resource permissions.
+
+[source, shell]
+----
+(Publisher) $ kubectl apply -f rbac.yaml
+----
+
+Then, update the WAN Replication configuration in publisher/hazelcast.yaml with the external IP of the Receiver cluster.
+
+[source, yaml]
+----
+wan-replication:
+  my-wan-replication:
+    batch-publisher:
+      my-publisher:
+        cluster-name: dev
+        target-endpoints: 35.240.109.13
+----
+
+Create ConfigMap with the Hazelcast configuration.
+
+[source, shell]
+----
+(Publisher) $ kubectl create configmap hazelcast-configuration --from-file=publisher/hazelcast.yaml
+----
+
+Again, we need to create a secret with the Hazelcast Enterprise license key.
 
-// Define the problem and tell reader what he/she will learn from this guide.
+[source, shell]
+----
+(Receiver) $ kubectl create secret generic hz-license-key --from-literal license=<hz-license-key>
+----
 
-== Prerequisites
+Finally, we can start the publisher Hazelcast cluster.
 
-// Define the prerequisites for the guide.
+[source, shell]
+----
+(Publisher) $ kubectl apply -f statefulset.yaml
+----
 
-////
-Write the steps in your guide here by making each step a different section
-////
+Your two Hazelcast clusters are set up with the WAN Replication. Now, we can check if everything works correctly.
 
-== Summary
+== 3. Verify WAN Replication
 
-// Provide a quick summary
+Insert data into the publisher Hazelcast cluster
 
-== See Also
+[source, shell]
+----
+(Publisher) $ kubectl exec -it hazelcast-0 -- /bin/bash
+# java -cp lib/hazelcast-enterprise-all*.jar com.hazelcast.client.console.ClientConsoleApp
+hazelcast[default] > ns rep
+namespace: rep
+hazelcast[rep] > m.put key value
+null
+----
 
-// Add some links to resources, such as other related guides.
-// Use relative links used on the home page (see https://raw.githubusercontent.com/hazelcast-guides/guides-site/master/home/modules/ROOT/pages/index.adoc)
+Check that the data was replicated to the receiver Hazelcast cluster.
 
-- xref:hazelcast-embedded-springboot:ROOT:index.adoc[Hazelcast in SpringBoot]
-- xref:hazelcast-quarkus:ROOT:index.adoc[Hazelcast Client for Quarkus] 
+[source, shell]
+----
+(Receiver) $ kubectl exec -it hazelcast-0 -- /bin/bash
+# java -cp lib/hazelcast-enterprise-all*.jar com.hazelcast.client.console.ClientConsoleApp
+hazelcast[default] > ns rep
+hazelcast[rep] > m.get key
+value
+----
```

Comment:
There might be a quick summary and "see also"s to other kubernetes guides - if they're not mentioned above.

Reply:
Done